bucket/networkmanager.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2021.1.0",
+    "description": "Tool for managing networks and network troubleshooting",
+    "homepage": "https://borntoberoot.net/NETworkManager/",
+    "license": "GPL-3.0-or-later",
+    "suggest": {
+        ".NET": "dotnet-sdk"
```

Comment:
why SDK when you need runtime?

Reply:
Because [`dotnet`](https://github.com/ScoopInstaller/Main/blob/master/bucket/dotnet.json#L6) manifest suggests to use `dotnet-sdk` instead, which also packs runtime. My removed comment was just about that...

Reply:
Why not suggest just runtime??

Reply:
Well because user (at least I) would search for `dotnet` in scoop official buckets and would install outdated version, which is not sufficient.

Reply:
So rather fill end-user's system with trash 200MB something, which they will never use instead of updating the runtime manifest?

Reply:
Yeah, it's not great, but I did not have time to fix that manifest (yet). IDK how to handle exe installers and there does not seem to be official archive version of installer.

Reply:
Only ASP.NET core and non desktop runtime (CLI only apps) seems to have zip archives. https://dotnet.microsoft.com/download/dotnet/5.0 . Looking into their contents, it should be possible to strip down sdk zip from non runtime components, but I can't see any easy way with scoop.

Reply:
https://github.com/lukesampson/scoop-extras/blob/master/bucket/windowsdesktop-runtime.json

Reply:
Well, that is unfortunate name. Yeah it is inherited from downloaded archive name but I'd still search for `dotnet` when I need to instal dotnet runtime. That is also reason why I was not aware of it.

Reply:
Would you be OK with new [dotnet-desktop-runtime manifest](https://gist.github.com/diggit/211b218071e759ed6c4b4402db256baf)? It has working auto update, it is installed systemlessly (in scoop apps) and IMO has better name.

Reply:
To overcome absence of archive for dotnet dekstop runtime, it uses SDK archive and removes all not necessary contents in install script. tested with NETworkManager and no problems occurred.

Reply:
No. You still need to download 200mb instead of like 50, which does not make sense.

Also all runtimes will be installed as vcredists and current desktop

Reply:
Well, so what should I put into suggest section for this manifest?

Reply:
Updated windowsdesktop-runtime with version 5

![image](https://user-images.githubusercontent.com/13260377/105609924-c2451f00-5dac-11eb-9e0e-0476038a388c.png)